### PR TITLE
DFK Subnet network information

### PR DESCRIPTION
**Pull request by magnusironroot**
This PR adds DFK Chain subnet details such as genesis block and deployed contract details.

### CODE REVIEW

File: networks/Readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Networks
```
Comment (patrick-ogrady) [inline networks/Readme.md:1]: This seems to unnecessarily duplicate this: https://github.com/ava-labs/subnet-evm/blob/master/networks/README.md

File: networks/mainnet/53935/Readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# DFK Chain Subnet
```
Comment (patrick-ogrady) [inline networks/mainnet/53935/Readme.md:1]: Should do `README.md`

File: networks/testnet/335/Readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# DeFi Kingdoms Fuji Subnet
```
Comment (patrick-ogrady) [inline networks/testnet/335/Readme.md:1]: Should do `README.md`

File: networks/Readme.md (hunk 1 above)
Comment (aaronbuchwald) [inline networks/Readme.md:1]: Can you make this change to `networks/README.md` instead and delete this file?

**Review by aaronbuchwald — APPROVED** [review]
LGTM - thanks for putting this together

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**